tests/gtest/aviftest_helpers.h

```diff
@@ -85,6 +91,8 @@ AvifImagePtr ReadImage(
         AVIF_CHROMA_DOWNSAMPLING_AUTOMATIC,
     avifBool ignore_icc = false, avifBool ignore_exif = false,
     avifBool ignore_xmp = false);
+// Convenient wrapper around avifPNGWrite() for debugging purposes.
+bool WriteImage(const avifImage* image, const char* file_path);
```

Comment:
I missed this new function. The function name should say the output file is a PNG file, e.g., "WritePNGImage" or "WriteImageToPNGFile".

Also, this function is not used.

Reply:
The function could be named `WritePNG()`. I just matched `ReadImage()` for consistency.
The function is not used because it exists for debugging purposes.

Reply:
"WritePNG" is a fine name. `ReadImage` doesn't specify the image format because it takes any image format as input (and fails if the input is of an unsupported format).

To prevent someone from removing it as an unused function, I suggest adding a comment (such as "Do not remove"), or replace "for debugging purposes" with something like "for temporary debugging code". (The temporary part is what I missed.)